src/app/(funnel)/foreign-language-cert/page.module.scss

```
@@ -0,0 +1,77 @@
+@use '@/styles/color.scss' as colors;
+@use '@/styles/radius.scss' as radius;
+@use '@/styles/spacing.scss' as spacing;
+@use '@/styles/typography.scss' as typography;
+@use '@/styles/utilities.scss' as utilities;
+
+.container {
+  display: flex;
+  flex-direction: column;
+  @include utilities.funnel-page-spacing;
+  // 고정 '다음' 버튼(fixed) 뒤로 마지막 카드가 가리지 않도록 하단 여유 확보.
```

Stylelint (17.13.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)